test/libsolidity/util/TestFileParser.cpp

```diff
@@ -489,11 +489,25 @@ string TestFileParser::parseString()
 	return literal;
 }
 
+namespace {
+string _readStream(istream& _stream)
+{
+        string ret;
+        while (!_stream.eof())
+        {
+                string tmp;
+                // NOTE: this will read until EOF or NL
+                getline(_stream, tmp);
+                ret.append(tmp);
+                ret.append("\n");
```

Comment:
The main change is that we reintroduce newlines, since `getline()` is stripping it. This is inspired by the implementation of `readStandardInput()`.